compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -3635,8 +3635,14 @@ TR::Register *OMR::Power::TreeEvaluator::vnegEvaluator(TR::Node *node, TR::CodeG
 
    switch(node->getDataType().getVectorElementType())
      {
+     case TR::Int8:
+       return TR::TreeEvaluator::vnegIntHelper(node,cg,TR::InstOpCode::vsububm,TR::InstOpCode::nop); //pass in nop since there is no single vector negate byte instruction
+     case TR::Int16:
+       return TR::TreeEvaluator::vnegIntHelper(node,cg,TR::InstOpCode::vsubuhm,TR::InstOpCode::nop); //pass in nop since there is no single vector negate halfword instruction
      case TR::Int32:
-       return TR::TreeEvaluator::vnegInt32Helper(node,cg);
+       return TR::TreeEvaluator::vnegIntHelper(node,cg,TR::InstOpCode::vsubuwm,TR::InstOpCode::vnegw);
+     case TR::Int64:
+       return TR::TreeEvaluator::vnegIntHelper(node,cg,TR::InstOpCode::vsubudm,TR::InstOpCode::vnegd);
```

Comment:
I think `TR::InstOpCode::bad` would work better than `nop` here since this is a placeholder and doesn't really have anything to do with `nop`.

It also might be better to just pass the data type into `vnegIntHelper`. It might be less confusing there and won't need to use placeholder opcodes at all.